.travis.yml

```diff
@@ -1,25 +1,31 @@
 language: scala
-scala:
-- 2.11.8
-- 2.12.1
-jdk:
-- oraclejdk8
+
 before_install:
 - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then bash
   scripts/decrypt-keys.sh; fi
 - export PATH=${PATH}:./vendor/bundle
+
 install:
 - rvm use 2.2.3 --install --fuzzy
 - gem update --system
 - gem install sass
 - gem install jekyll -v 3.2.1
-script:
-- sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport
-- sbt ++$TRAVIS_SCALA_VERSION "docs/tut"
 
 after_success:
 - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
     sbt ++$TRAVIS_SCALA_VERSION publishSnapshot;
     bash scripts/publishMicrosite.sh;
   fi
-- bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+- bash <(curl -s https://codecov.io/bash)
+
+matrix:
+  include:
+   - scala: 2.11.8
+     jdk: oraclejdk8
+     script:
+      - sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport
+      - sbt ++$TRAVIS_SCALA_VERSION "docs/tut"
+   - scala: 2.12.1
+     jdk: oraclejdk8
+     script:
+      - sbt ++$TRAVIS_SCALA_VERSION clean tests/test
```

Comment:
Travis file looks good, however:

* For scala 2.12, we are only running this test: https://github.com/47deg/freestyle/blob/master/tests/src/test/scala/test.scala . Maybe changing `sbt ++$TRAVIS_SCALA_VERSION clean tests/test` by `sbt ++$TRAVIS_SCALA_VERSION clean test` would be better? Or maybe creating a test suite excluding the play tests...
* Can we run `tut` also for scala 2.12?